addons/slideshow/functions/fnc_autoTransition.sqf

```diff
@@ -18,7 +18,7 @@
  * Public: No
  */
 
-params ["_objects", "_images", "_varString", "_duration"];
+params ["_objects", "_images", "_varString", "_currentSlideshow", "_duration"];
```

Comment:
header needs to be updated with new parameters

Reply:
Function header updated: 38d4c90eef4aa2b16d284318217ae0186eb943f3